docs/guides/primitives.mdx

```diff
@@ -109,9 +77,9 @@ estimator.run([(circuit1, observable1, param_values1),(circuit2, observable2, pa
 ```
 
 <span id="broadcast-rules"></span>
-##### Broadcasting rules
+### Broadcasting rules
```

Comment:
Do we need broadcasting rules here given they are repeated in the `Primitive inputs and outputs` page?

Reply:
No!  good catch!